environment.yml

```
@@ -20,6 +21,8 @@ dependencies:
   - tofu
   - omas
   - xbout
+  - omfit-opt-deps
+  - omfit_classes
   - pip:
     - turbopy
```

```suggestion
    - gpr1dfusion
    - turbopy
```
My best guess is that having it installed by `pip` would suffice.

---

Good try, but this will still fail the conda solve.  I have added the correct channel that is needed (`aaronkho`).